lib/graphql/schema.rb

```diff
@@ -137,6 +137,7 @@ def initialize
       @subscription_execution_strategy = self.class.default_execution_strategy
       @default_mask = GraphQL::Schema::NullMask
       @rebuilding_artifacts = false
+      @trace_scalar_fields_by_default = false
```

Comment:
Can this ivar be removed now?

Reply:
Yup! Sorry, looks like I forgot about this one.